docs/1.getting-started/6.data-fetching.md

```diff
@@ -127,6 +127,56 @@ watch(count, (newCount) => {
 </script>
 ```
 
+
+## Transforming Data
+
+If for some reason you are not satisfied with how the requested data is structured, you may want to use the `transform` option from `useFetch` or `useAsyncData` to alter the data by using a function to perform the necessary transformation after your request has been resolved.
```

Comment:
Would it make sense to include this in [this part of the page](https://nuxt.com/docs/getting-started/data-fetching#minimize-payload) - or link between them somehow?

Reply:
Good point. Did not notice the already existing example on the `pick` option in the Best Practices section. I added a link to this section pointing the Minimize Payload explanation.